src/tm_devices/device_manager.py

```diff
@@ -284,6 +283,8 @@ def __init__(
         self.__config = DMConfigParser()
         if config_options is not None:
             self.__config.update_options(config_options)
+        if self.__config.options.check_for_updates:  # pragma: no cover
```

Comment:
This changes the default behavior to no longer check for updates. Is that an acceptable change?

Reply:
I think that'll be fine